packages/extension/src/shared/utils/fetchImmutable.ts

```diff
@@ -20,6 +20,14 @@ export async function fetchImmutable<Data>(key: string, fetcher: Fetcher<Data, s
   }
 }
 
+export async function getImmutable<Data>(key: string): Promise<Data | undefined> {
+  return await fetchImmutableStorage.get(key)
+}
+
+export async function storeImmutable<Data>(key: string, data: Data): Promise<void> {
+  return await fetchImmutableStorage.set(key, data)
+}
```

Comment:
Local storage has size constraints, so if the cache grows too large, it can consume all available local storage.

This is something to be aware of, although we can improve it later.